openapi/featurestore.yaml

```diff
@@ -404,11 +415,14 @@ components:
           type: object
           additionalProperties:
             type: string
+        description:
+          type: string
       required:
         - name
         - features
         - entities
         - tags
+        - description
```

Comment:
for description being required, are we going to pass in an empty one if the user doesn't define one?

Reply:
I checked feast FeatureView implementation, although it's optional when defining a FeatureView, feast will set the value to be an empty string if not provided. So when we are reading a FeatureView object, that field is always there.